packages/config/config/devices/0x010f/fgd211.json

```diff
@@ -230,6 +231,32 @@
 				}
 			]
 		},
+		{
+			"#": "14",
+			"$if": "firmwareVersion >= 2.2",
+			"label": "Inputs Button/Switch configuration",
+			"description": "Binary inputs type configuration",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
```

Comment:
Please change this to 2 and for the <=2.1 variant to 1. This should match the options.

Reply:
Thanks for the suggestion. I have updated it